dummy_data_generation/schemas.py

```diff
@@ -6,6 +6,24 @@
 yes_no_unknown_choice = ["Yes", "No", "Don't know", None]
 people_count = ["0", "1 to 5", "6 to 10", "11 to 20", "21 or more", "Don't know", "Prefer not to say", None]
 times_count = ["1", "2", "3", "4", "5", "6", "7 times or more", "Don't know", "None", "Prefer not to say", None]
+consolidation_points = [
```

Comment:
consolidation_points will be pre-cleaned by IQVIA, so can remove the 'Consolidation' prefix from all values.